### PR TITLE
CSS: rename some variables to be consistent

### PR DESCRIPTION
**Pull request by louismaximepiton**
### Description

Rename each `-2xl` to `-xxl`.

### Motivation & Context

Be consistent with other variables.

### Type of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring (non-breaking change)
- [ ] Breaking change (fix or feature that would change existing functionality)

### Checklist

- [x] I have read the [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md)
- [x] My code follows the code style of the project _(using `npm run lint`)_
- [x] My change introduces changes to the documentation
- [x] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests passed

#### Live previews

- <https://deploy-preview-37804--twbs-bootstrap.netlify.app/>

### Related issues

NA

### CODE REVIEW

**Comment by louismaximepiton** [comment]
Is there any way to deprecate those CSS variables ?

**Comment by julien-deramond** [comment]
Haven't reviewed precisely the PR but I agree with the principle here for consistency through the whole framework.

@twbs/css-review I'm not too familiar with when we authorize to deprecated some Sass vars like this. I'd say that in this case, it would be OK  to deprecate them in v5.3 and not wait until v6. But a second opinion would help me :)

> Is there any way to deprecate those CSS variables ?

Right now, I believe that only our migration guide do the job

**Comment by mdo** [comment]
This is an interesting one because one request we had late in v5's development to consider for v6 is that we use `2xl` instead of `xxl` as it might scale better to even larger sizes like `3xl`, `4xl`, etc. I hate the inconsistency, but maybe we could reassign the old CSS and Sass vars to the new value, and then back track in v6? Thoughts?

**Comment by julien-deramond** [comment]
> consider for v6 is that we use 2xl instead of xxl as it might scale better to even larger sizes like 3xl, 4xl, etc.

Oh yeah nice, it will be better like this indeed.

> I hate the inconsistency, but maybe we could reassign the old CSS and Sass vars to the new value, and then back track in v6? Thoughts?

Works for me, sounds like a good strategy

File: scss/_root.scss [hunk 1]
```diff
@@ -106,7 +106,7 @@
   --#{$prefix}border-radius-sm: #{$border-radius-sm};
   --#{$prefix}border-radius-lg: #{$border-radius-lg};
   --#{$prefix}border-radius-xl: #{$border-radius-xl};
-  --#{$prefix}border-radius-2xl: #{$border-radius-2xl};
+  --#{$prefix}border-radius-xxl: #{$border-radius-xxl};
```
Comment (mdo) [inline scss/_root.scss:109]:
```suggestion
  --#{$prefix}border-radius-xxl: #{$border-radius-xxl};
  --#{$prefix}border-radius-2xl: var(--#{$prefix}border-radius-xxl); // Deprecated in v5.3.0 for consistency
```

File: scss/_variables.scss [hunk 2]
```diff
@@ -535,9 +535,12 @@ $border-radius:               .375rem !default;
 $border-radius-sm:            .25rem !default;
 $border-radius-lg:            .5rem !default;
 $border-radius-xl:            1rem !default;
-$border-radius-2xl:           2rem !default;
+$border-radius-xxl:           2rem !default;
 $border-radius-pill:          50rem !default;
 // scss-docs-end border-radius-variables
+// fusv-disable
+$border-radius-2xl:           2rem !default; // Deprecated in 5.3.0
```
Comment (mdo) [inline scss/_variables.scss:542]:
```suggestion
$border-radius-2xl:           $border-radius-xxl !default; // Deprecated in v5.3.0
```

**Review by mdo — COMMENTED** [review]
I think these two comments address it?

**Pull request merged**